### ISSUE TITLE
ionic 4.3.0 command fail due to  missing `IONIC_CONFIG_DIRECTORY`

### ISSUE THREAD

**Issue by brotich** [ionic-team/ionic-cli#3724]
**Description:**

Building ios application on ionic fails due to missing config directory

**Steps to Reproduce:**

1. install ionic cli 4.3.0
```bash
yarn global add ionic@4.3.0
```
1. from an ionic project directory, run the command
```bash
ionic cordova platform add ios --no-resources
```

**Output:**

```
bash-3.2$ ionic --version
Cannot read property 'IONIC_CONFIG_DIRECTORY' of undefined
```

**My `ionic info`:**

```
Cannot read property 'IONIC_CONFIG_DIRECTORY' of undefined
```

**Other Information:**

- using version 4.1.2 does not display the error above

**Comment by dwieeb** [comment]
@brotich What version of Node are you using? This error would imply that `process.env` is undefined. Any reason why that might be?

**Comment by dwieeb** [comment]
@brotich Do you have `ionic` installed locally? If so, try uninstalling it.

**Comment by brotich** [comment]
> What version of Node are you using? This error would imply that process.env is undefined.

Node version `v10.11.0`

> @brotich Do you have ionic installed locally? If so, try uninstalling it.

with the assumption that  locally means in the `node_modules`, i have ionic installed in the globally via `yarn global add ionic`

cc: @dwieeb

**Comment by dwieeb** [comment]
@brotich Yes, within `node_modules`. Could you check?

```
ls node_modules/ionic
```

If it does not exist, I'm not sure what could be going wrong. Perhaps try reinstalling it?

**Comment by dwieeb** [comment]
@brotich Yeah, it's interfering. I must've missed something in the latest update.

The workaround is to match the global/local versions OR uninstall the local version, unless you need it for some reason:

```
npm uninstall ionic
```

**Comment by dwieeb** [comment]
@brotich Will be fixed in 4.3.1. I need to fix another issue before release

**Comment by dwieeb** [comment]
Thank you for the bug report!!
